assets/js/blocks/checkout/cart/index.js

```diff
@@ -17,6 +17,12 @@ registerBlockType( 'woocommerce/checkout-cart', {
 		html: false,
 	},
 	edit() {
+		const { hasShippingEnabled, activeShippingMethods } = wc_checkout_block_data;
+
+		const methods = Object.keys( activeShippingMethods ).map( method => {
+			return <li>{ activeShippingMethods[ method ].title }</li>;
```

Comment:
I think we should add an input radio before the input name even if it's just a disabled input.
<img width="530" alt="Screen Shot 2019-06-06 at 4 02 02 PM" src="https://user-images.githubusercontent.com/10561050/59039091-73afdd80-8874-11e9-9f96-8087aceea5b3.png">

Would also be nice to add a price after the name, but we can do in a follow-up PR if that's possible.